.circleci/config.yml

```diff
@@ -171,6 +171,20 @@ jobs:
           no_output_timeout: 30m
       - upload_artifacts
 
+  server-wallet-simple-stress-test:
+    <<: *postgres
+    steps:
+      - checkout
+      - attach_workspace:
+          at: /home/circleci/project
+      - install_postgresql_client
+      - run:
+          name: server-wallet-simple-stress-test
+          working_directory: packages/server-wallet
+          command: npx ts-node ./e2e-testing/scripts/start-all.ts -l ./e2e-testing/test-data/simple-ci-run.json
+          no_output_timeout: 30m
```

Comment:
This is a really large value. What's the purpose of a 30m timeout here?

Reply:
This was copied from the existing `server-wallet-with-peers-test` job. It looks like all jobs currently have `     no_output_timeout: 30m` ?

Reply:
I see! Might be worth using the default of 10m, unless there's a good reason for that?